### DISCUSSION TITLE
The reduce operator could benefit from an additional suffix

### DISCUSSION THREAD

**Discussion by BurningTreeC** [TiddlyWiki/TiddlyWiki5#5944] [Ideas]
As the title says, I believe that the `reduce` operator would benefit from an additional suffix which defines the variable name where the result is stored

```
:reduce:myVar[...]
```

**Comment by saqimtiaz** [comment]
Do you mean the filter run prefix, or the operator?

Can you provide some examples of use cases that cannot currently be accomplished, which would be facilitated by such a change?

**Reply by BurningTreeC** [reply]
oh sorry, I mean the filter run prefix

an example would be if I want to calculate all of this in one filter run:
- the sum of some variables and values I get from fields
- the sum of other variables and values I get from fields
- after that, the difference between the two
